esphome/core/util.cpp

```diff
@@ -94,6 +94,13 @@ void network_setup_mdns(IPAddress address, int interface) {
       MDNS.addServiceTxt("esphomelib", "tcp", "version", ESPHOME_VERSION);
       MDNS.addServiceTxt("esphomelib", "tcp", "address", network_get_address().c_str());
       MDNS.addServiceTxt("esphomelib", "tcp", "mac", get_mac_address().c_str());
+#ifdef ARDUINO_ARCH_ESP8266
+      MDNS.addServiceTxt("esphomelib", "tcp", "platform", "ESP8266");
+#endif
+#ifdef ARDUINO_ARCH_ESP32
+      MDNS.addServiceTxt("esphomelib", "tcp", "platform", "ESP32");
+#endif
+      MDNS.addServiceTxt("esphomelib", "tcp", "board", ESPHOME_BOARD);
```

Comment:
```suggestion
      MDNS.addServiceTxt("esphomelib", "tcp", "board", ESPHOME_BOARD);
```

The ESPHOME_BOARD define needs to be added to `esphome/core/defines.h` too (it's a dummy file to make cpp syntax highlighting work for vscode etc).

Reply:
Thanks